Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -1988,7 +1988,7 @@ void MainWindow::showTTErrorMessage(const ClientErrorMsg& msg, CommandComplete c
                 return;
             
             addLatestHost(m_host);
-            QString nickname = ttSettings->value(SETTINGS_GENERAL_NICKNAME, QCoreApplication::translate("MainWindow", SETTINGS_GENERAL_NICKNAME_DEFAULT)).toString();
+            QString nickname = ttSettings->value(SETTINGS_GENERAL_NICKNAME).toString();

```

Comment:
Keep SETTINGS_GENERAL_NICKNAME_DEFAULT but set SETTINGS_GENERAL_NICKNAME_DEFAULT to an empty string